src/main/java/com/revature/lostchapterbackend/controller/AuthenticationController.java

```diff
@@ -46,7 +46,7 @@
 			
 			Users user = this.us.createUser(dto);
 			
-			return ResponseEntity.status(201).body(user);
+			return ResponseEntity.status(201).body("Successfully Sign up");
```

Comment:
As per frontend's request, changing the response body into a text instead of the Users object